src/CSF.SRDashboard.Client/Pages/SeafarerProfile.razor.cs

```diff
@@ -45,10 +63,26 @@ public partial class SeafarerProfile
 
         public string currentRelativePath;
 
-        
+        private int RequestID;
+        [Inject]
+        public IWorkLoadManagementService WorkLoadService { get; set; }
+
+
+
+        public List<WorkloadRequestTableItem> tableItems;
+
 
         protected async override Task OnInitializedAsync()
         {
+            if (requestId != 0)
+            {
+                IsAlertEnabled = true;
+            }
+            else
+            {
+                IsAlertEnabled = false;
+            }
+
```

Comment:
I'm assuming it's a class  variable, but for readability we probably want to preface it with this. This line can be simplified to
```suggestion
            this.IsAlertEnable = requested != 0;
```

Reply:
Changed